src/map/pc.c

```diff
@@ -8417,8 +8417,12 @@ static int64 pc_readparam(const struct map_session_data *sd, int type)
 		case SP_FLEE1:           val = sd->battle_status.flee; break;
 		case SP_FLEE2:           val = sd->battle_status.flee2; break;
 		case SP_DEFELE:          val = sd->battle_status.def_ele; break;
-#ifndef RENEWAL_CAST
 		case SP_VARCASTRATE:
+#ifdef RENEWAL_CAST
+			val = sd->bonus.varcastrate;
+			break;
+#else
+			FALLTHROUGH
```

Comment:
Is this correct what SP_VARCASTRATE on pre renewal will works same as SP_CASTRATE?
Before SP_VARCASTRATE on pre renewal was ignored.

Reply:
It's the same as before. The old code:
```
#ifndef RENEWAL_CAST
		case SP_VARCASTRATE:
#endif
		case SP_CASTRATE:
				val = sd->castrate;
			break;
```
If `RENEWAL_CAST` is not defined, `SP_VARCASTRATE` falls through to `SP_CASTRATE`.
I just explicitly stated the `FALLTHROUGH` for better readability,

Reply:
FALLTHROUGH here need for mark what missing break is not at issue. This need for compiler